arangod/Cluster/Utils/ShardID.h

```diff
@@ -0,0 +1,161 @@
+////////////////////////////////////////////////////////////////////////////////
+/// DISCLAIMER
+///
+/// Copyright 2023-2023 ArangoDB GmbH, Cologne, Germany
+///
+/// Licensed under the Apache License, Version 2.0 (the "License");
+/// you may not use this file except in compliance with the License.
+/// You may obtain a copy of the License at
+///
+///     http://www.apache.org/licenses/LICENSE-2.0
+///
+/// Unless required by applicable law or agreed to in writing, software
+/// distributed under the License is distributed on an "AS IS" BASIS,
+/// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+/// See the License for the specific language governing permissions and
+/// limitations under the License.
+///
+/// Copyright holder is ArangoDB GmbH, Cologne, Germany
+///
+/// @author Michael Hackstein
+////////////////////////////////////////////////////////////////////////////////
+
+#pragma once
+
+#include "Basics/Exceptions.h"
+#include "Basics/ResultT.h"
+#include "Basics/StringUtils.h"
+#include "Inspection/Status.h"
+
+#include <compare>
+#include <cstdint>
+#include <fmt/format.h>
+#include <velocypack/Value.h>
+
+namespace arangodb {
+
+struct ShardID {
+
+  static ResultT<ShardID> shardIdFromString(std::string const& s) {
```

Comment:
Additionally, the header `<string_view>` should be included here.
```suggestion
  static ResultT<ShardID> shardIdFromString(std::string_view s) {
```

Reply:
Done